.github/workflows/ci.yml

```diff
@@ -61,22 +61,30 @@ jobs:
         # Install solc unconditionally because it only takes a second or two
         sudo wget -O /usr/bin/solc https://github.com/ethereum/solidity/releases/download/v0.4.24/solc-static-linux
         sudo chmod +x /usr/bin/solc
+        pip install coveralls
         pip install -e ".[dev-noks]"
     - name: Run Tests
       env:
         TEST_TYPE: ${{ matrix.type }}
       run: |
         cp scripts/run_tests.sh .
         ./run_tests.sh
-    - name: Coverage Upload
-      uses: codecov/codecov-action@v1
+    - name: Coveralls Parallel
```

Comment:
Is it necessary to manually install and run the CLI tool? Or can we use the GH action as shown in the example here: https://github.com/marketplace/actions/coveralls-github-action#complete-parallel-job-example

Reply:
Good question. I couldn't get it to work since it expects a `.lcov` file.

Here's an open issue if you can find a better workaround https://github.com/coverallsapp/github-action/issues/30